Fix raise AssertionError in test_half.py

VictorRodriguez opened the pull request:
Patch fix raise AssertionError(smsg)
AssertionError: Did not raise floating point underflow error

Comming from:

test_half_fpe float16(2.**-14), float16(2**11))
File
"/usr/lib/python2.7/site-packages/numpy/core/tests/test_half.py",
line 19, in assert_raises_fpe
"Did not raise floating point %s error" % strmatch)
File "/usr/lib/python2.7/site-packages/numpy/testing/utils.py", line 92, in
assert

assert_ was designed for the cases where python optimized is
executed, however in general cases this is not working as espected
raising errors

Signed-off-by: Victor Rodriguez <victor.rodriguez.bahena@intel.com>

VictorRodriguez committed:
Fix raise AssertionError in test_half.py

Patch fix raise AssertionError(smsg)
AssertionError: Did not raise floating point underflow error

Comming from:

test_half_fpe float16(2.**-14), float16(2**11))
File
"/usr/lib/python2.7/site-packages/numpy/core/tests/test_half.py",
line 19, in assert_raises_fpe
"Did not raise floating point %s error" % strmatch)
File "/usr/lib/python2.7/site-packages/numpy/testing/utils.py", line 92, in
assert

assert_ was designed for the cases where python optimized is
executed, however in general cases this is not working as espected
raising errors

Signed-off-by: Victor Rodriguez <victor.rodriguez.bahena@intel.com>

charris commented:
This doesn't look right at all. What is the problem with `assert_`, and why give the assert statement a tuple?

VictorRodriguez committed:
Fix ValueError in test_arraypad.py

In test_check_padding_an_empty_array

a = pad(np.zeros((0, 3)), ((0,), (1,)), mode='reflect')

generate an array of 0 raws, this generate a raise ValueError in arraypad.py:

    if narray.size == 0:

This is fixed generating an array of 1 raw and 3 columns

Signed-off-by: Victor Rodriguez <victor.rodriguez.bahena@intel.com>

charris commented:
Also, do not work in master, make your pull requests from a branch.

VictorRodriguez commented:
Sorry my mistake , please close the pull request I will send a new one properly
